### PR TITLE
Replace `julia` with Julia in faq when referring to the language.

### PR DESCRIPTION
**Pull request by LilithHafner**
https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-julia-code-contributions
> in docstring refer to the language as "Julia" and the executable as "julia"

### CODE REVIEW

File: doc/src/manual/faq.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ On the other hand, language *interoperability* is extremely useful: we want to e
 
 ### How does Julia define its public API?
 
-The only interfaces that are stable with respect to [SemVer](https://semver.org/) of `julia`
+The only interfaces that are stable with respect to [SemVer](https://semver.org/) of Julia
```
Comment (ViralBShah) [inline doc/src/manual/faq.md:25]: The language feels a bit weird here. Not your edits, but the sentence originally. Anything you can do to craft it better?

Comment (LilithHafner) [reply doc/src/manual/faq.md:25]: Well... I propose to totally write this section in https://github.com/JuliaLang/julia/pull/50105, but the scope of that is way larger than this.

Comment (LilithHafner) [reply doc/src/manual/faq.md:25]: How does this sound?
 > Base and standard library functionality described in [the documentation](https://docs.julialang.org/), which is not labeled as unstable (such as experimental and internal), is covered by [SemVer](https://semver.org/)." Functions, types, and constants are not part of the public API if they are not included in the documentation, _even if they have docstrings_.

Comment (LilithHafner) [reply doc/src/manual/faq.md:25]: These are separate changes. I'm going to merge this as is, and do the reword in another PR

**Pull request merged**